Dockerfile

```diff
@@ -0,0 +1,24 @@
+# renovate: datasource=docker depName=php
+ARG PHP_VERSION=8.3.2
+
+# renovate: datasource=docker depName=alpine
+ARG ALPINE_VERSION=3.18
+
+FROM php:${PHP_VERSION}-fpm-alpine${ALPINE_VERSION}
```

Comment:
I don't think it should require `fpm`, the CLI context should be fine

Reply:
done